.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+
+      - name: Install yarn
```

Comment:
is there an action that already has yarn? this likely adds more time than needed to CI

Reply:
There is this https://github.com/Borales/actions-yarn

I have updated the code to use this action. My only concern is that we won't have the `build/dist` directory available when pushing to npm registry. Not sure if we lose our place in the server, when using different github actions 🤔

Reply:
I see, right. these aren't like docker images, where you have some env setup. Maybe it does make sense then to use it like you had it? Might be worth checking out how we do this in react-bootstrap: https://github.com/react-bootstrap/react-bootstrap/blob/master/.github/workflows/nodejs.yml

Reply:
Ok thanks for sharing the link, it looks like we never needed to install yarn since `actions/setup-node@v1` does it for us. What waste of time 🤦‍♂️

Appreciate all your comments. The PR looks good now, much cleaner.

Let's put it to test!

Reply:
Don't forgot to setup the `NPM_TOKEN` key in the settings